tests/perf-test/SearchPage.perf-test.js

```diff
@@ -0,0 +1,239 @@
+import {act, fireEvent, screen} from '@testing-library/react-native';
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import {measurePerformance} from 'reassure';
+import _ from 'underscore';
+import SearchPage from '@pages/SearchPage';
+import ComposeProviders from '../../src/components/ComposeProviders';
+import {LocaleContextProvider} from '../../src/components/LocaleContextProvider';
+import OnyxProvider from '../../src/components/OnyxProvider';
+import {CurrentReportIDContextProvider} from '../../src/components/withCurrentReportID';
+import {KeyboardStateProvider} from '../../src/components/withKeyboardState';
+import {WindowDimensionsProvider} from '../../src/components/withWindowDimensions';
+import CONST from '../../src/CONST';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import createPersonalDetails from '../utils/collections/personalDetails';
+import createRandomReport from '../utils/collections/reports';
+import PusherHelper from '../utils/PusherHelper';
+import * as TestHelper from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+import wrapOnyxWithWaitForBatchedUpdates from '../utils/wrapOnyxWithWaitForBatchedUpdates';
+
+jest.mock('../../src/libs/Navigation/Navigation');
+
+const mockedNavigate = jest.fn();
+jest.mock('@react-navigation/native', () => {
+    const actualNav = jest.requireActual('@react-navigation/native');
+    return {
+        ...actualNav,
+        useFocusEffect: jest.fn(),
+        useIsFocused: () => ({
+            navigate: mockedNavigate,
+        }),
+        useRoute: () => jest.fn(),
+        useNavigation: () => ({
+            navigate: jest.fn(),
+            addListener: () => jest.fn(),
+        }),
+        createNavigationContainerRef: jest.fn(),
+    };
+});
+
+const getMockedReportsMap = (length = 100) => {
+    const mockReports = Array.from({length}, (__, i) => {
+        const reportID = i + 1;
+        const report = createRandomReport(reportID);
+        const reportKey = `${ONYXKEYS.COLLECTION.REPORT}${reportID}`;
+
+        return {[reportKey]: report};
+    });
+
+    return _.assign({}, ...mockReports);
+};
+
+const getMockedPersonalDetailsMap = (length) => {
```

Comment:
Instead of creating  `getMockedPersonalDetailsMap` we can use `createCollection`. Take a look at  [docs](https://github.com/Expensify/App/blob/978b39f785431d8ee37589906076e50c3261835a/tests/README.md?plain=1#L18. ) . Same with `getMockedReportsMap`